src/test/java/org/folio/service/EcsTlrServiceTest.java

```diff
@@ -123,4 +127,29 @@ void canNotCreateRemoteRequestWhenFailedToPickTenant() {
 
     assertEquals("Failed to pick tenant for instance " + instanceId, exception.getMessage());
   }
+
+  @Test
+  void canCreateRemoteRequestOnlyForSecondTenantId() {
+    String instanceId = UUID.randomUUID().toString();
+    EcsTlr ecsTlr = new EcsTlr()
+      .instanceId(instanceId)
+      .id(UUID.randomUUID().toString());
+    String firstTenantId = UUID.randomUUID().toString();
+    String secondTenantId = UUID.randomUUID().toString();
+    String thirdTenantId = UUID.randomUUID().toString();
+
+    List<String> mockTenantIds = List.of(
+      firstTenantId, secondTenantId, thirdTenantId);
+    when(tenantPickingStrategy.pickTenants(any(String.class)))
+      .thenReturn(mockTenantIds);
+    when(tenantScopedExecutionService.execute(any(), any()))
+      .thenThrow(new TenantScopedExecutionException(new RuntimeException("Test failure"), firstTenantId))
+      .thenReturn(new Request().id(UUID.randomUUID().toString()))
+      .thenReturn(new Request().id(UUID.randomUUID().toString()));
+    ecsTlrService.create(ecsTlr);
+
+    ArgumentCaptor<EcsTlrEntity> captor = ArgumentCaptor.forClass(EcsTlrEntity.class);
```

Comment:
It might  be easier to just check tenantId in the result of `ecsTlrService.create(ecsTlr)`